static/sass/_pattern_shop-cart.scss

```diff
@@ -12,11 +12,22 @@
       display: flex;
       flex-wrap: wrap;
 
+      > .p-label {
+        align-self: center;
+        margin-top: 0.5rem;
+        margin-left: 0.5rem;
+      }
```

Comment:
Unless there already exists a set of utility classes like `u-no-margin--top` that could help you achieve the same result without having to declare your own styles, that would work as well (as it would decrease selectors complexity, too). Not sure which approach would be more in-line with Vanilla. What do you think?